test/framework/integration_framework/integration_test_framework.cpp

```diff
@@ -96,7 +98,11 @@ namespace integration_framework {
           commit_observable.subscribe([this](auto committed_block) {
             block_queue_.push(
                 std::make_shared<iroha::model::Block>(committed_block));
+            log_->info("block");
+            queue_cond.notify_all();
```

Comment:
Why is that the same `cv` that is used for a proposal? I think they should be different

Reply:
E.g in theory there can be multiple proposals without the commits but `checkProposal(..).checkBlock(..)` will succeed

Reply:
@l4l as `checkProposal` and `checkBlock` are called sequentially in one thread there are no reason to use different `cv` as they going to block each other any way